base/src/main/java/io/spine/validate/Duplicates.java

```diff
@@ -20,42 +20,30 @@
 
 package io.spine.validate;
 
-import io.spine.validate.option.FieldValidatingOption;
-import io.spine.validate.option.ValidatingOptionFactory;
+import com.google.common.collect.ImmutableSet;
 
+import java.util.Collection;
+import java.util.HashSet;
 import java.util.Set;
 
-import static java.lang.Math.abs;
-
-/**
- * Validates fields of {@link Float} types.
- */
-class FloatFieldValidator extends FloatFieldValidatorBase<Float> {
+public final class Duplicates {
```

Comment:
From what I see the sole method of this utility class is not used. Please check and eliminate if it is indeed not used. Otherwise, please document.